priv/repo/migrations/20230105144433_create_session_stats_partitions_through_2030.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Ret.Repo.Migrations.CreateSessionStatsPartitionsThrough2030 do
+  use Ecto.Migration
+
+  import Ret.Repo.MigrationHelpers, only: [next_month: 2]
+
+  @max_year 2030
+
+  def up do
+    for y <- 2023..@max_year,
+        m <- 1..12 do
+      next_month = next_month(y, m)
+
+      execute """
+      CREATE TABLE IF NOT EXISTS ret0.session_stats_y#{y}_m#{m} PARTITION OF ret0.session_stats
```

Comment:
Does the original migration already cover us up to 2030? https://github.com/mozilla/reticulum/blob/0f94ced4785f1b24e2236d4c3fbe797d2e8e7f38/priv/repo/migrations/20180408231444_create_session_stats_table.exs#L4

Reply:
This is precisely why changes should never be made to the logic of an existing migration after that migration has been deployed

Reply:
Ooh, I see. Thanks!